### ISSUE TITLE
[Unstable] Cost and scaling of Submarine upgrades

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#9831]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Every sub upgrade besides Engine/Pumps starts at around or under 2000mk, which means you can purchase one or several lvl 1 upgrades on the first outpost. (also see  #9822 )

![image](https://user-images.githubusercontent.com/73229309/188457746-ea4d2ac7-02f7-4323-b543-ca50c1994718.png)

Upgrade cost formula then skyrockets to insane values, even reaching >100k for lvl6 Hull

![image](https://user-images.githubusercontent.com/73229309/188456596-34175abc-63c9-4a18-a9ff-51a12f857604.png)

_Assuming the formula is Baseprice x ((1.5 + (tier x 0.25))_

In short:
**Base price is too low, but scaling is too sharp**. 
You shouldnt be able to buy sub upgrades as soon as the first outpost of the first biome. 
Increase base upgrade costs to at least 5000mk (if not more), but make the scaling less aggressive (all upgrade costs should be in the 5-20k ballpark)

### Version

0.19.3.0 (unstable)

**Comment by Ek-Videogames** [comment]
I don't really see 2,000mk price as a 'bug' considering you have to invest into many - many - levels before getting a noticeable benefit. I saw the old upgrade balancing as "pay as you go" for small upgrades over time rather than saving it all up for one big upgrade.

If the starting price were raised to say 4,000mk - even with no exponential price increase, 
that would still be a total of 24,000mk for a level-6 upgrade.

Idea:
- Higher upgrade staring cost to satiate bizarre inaccessibility OCD.
- Squeeze upgrades into fewer, larger increments.

Though I hardly know where to start with price economy seeing as how things are kind of in balance limbo atm.

**Comment by Rokvach** [comment]
No issues, closing.
